Cargo.toml

```diff
@@ -19,28 +19,32 @@ duct = "0.13"
 git2 = "0.18"
 tempfile = "3.8"
 tokio = { version = "1.0", features = ["macros", "rt-multi-thread"] }
-url = { version = "2.5"}
+url = { version = "2.5" }
```

Comment:
Can we sort these alphabetically please? Makes it easier to visually scan to find something, especially when on mobile and ctrl-f not as easy.

Rest of the file should be done, but out of scope of this PR